server/sonar-web/src/main/js/components/controls/FavoriteBase.tsx

```diff
@@ -80,13 +82,18 @@ export default class FavoriteBase extends React.PureComponent<Props, State> {
   }
 
   render() {
+    const tooltip = this.state.favorite
+      ? translate('favorite.current')
+      : translate('favorite.check');
     return (
-      <a
-        className={classNames('display-inline-block', 'link-no-underline', this.props.className)}
-        href="#"
-        onClick={this.toggleFavorite}>
-        <FavoriteIcon favorite={this.state.favorite} />
-      </a>
+      <Tooltip overlay={tooltip}>
```

Comment:
The tooltip should be placed on the side, because right now it's not correctly placed.